lib/logger.js

```diff
@@ -2,7 +2,7 @@ var winston = require('winston');
 
 var logger = new (winston.Logger)({
     transports: [
-        new (winston.transports.Console)({'timestamp':true})
+        new (winston.transports.Console)()
```

Comment:
Those brackets look weird to me.  Would `new winston.transports.Console()` be incorrect?